traffic_ops/testing/api/v3/tc-fixtures.json

```diff
@@ -1543,6 +1543,10 @@
         {
             "xmlId": "test-ds-server-assignments",
             "serverNames": ["test-ds-server-assignments"]
+        },
+        {
+            "xmlId": "ds3",
+            "serverNames": ["atlanta-edge-14"]
```

Comment:
why'd you add this?

Reply:
This test: `UpdateDSGetServerDSID` needs it as a prerequisite.

Reply:
right, but that dependency was also added by this PR, so then my question becomes: "Why require a new Delivery Service Servers instead of using an existing one?"

Reply:
The existing ones do not work for that test and they are also being used for other server tests.

Reply:
> The existing ones do not work for that test

I can't see why at a glance, but I'll take your word for it.